apps/client/src/common/components/timer-display/TimerDisplay.tsx

```diff
@@ -19,12 +19,18 @@ interface TimerDisplayProps {
 const TimerDisplay = (props: TimerDisplayProps) => {
```

Comment:
Looks like this component is now only used in PlaybackTimer, perhaps we have a good chance to either move the logic there and delete this or tighten the implementation to only have what is necessary. No need to think about reusability if we decide not to reuse it